GitUI/Script/FilePrompt.cs

```diff
@@ -25,15 +26,29 @@ private void FilePrompt_Shown(object sender, EventArgs e)
 
         private void btn_Browse_Click(object sender, EventArgs e)
         {
+            StringBuilder sb = new StringBuilder();
+            string separator = "; ";
```

Comment:
Allow configuration of the separator.

Reply:
Where and how?

Reply:
Add the functionality so that the user of the prompt can set the separator.

Turn it into a property of the class and put in validation to prevent empty strings.
private string separator;

		  public string Separator
		{
				get => separator;
				set
				{
					if (!string.IsNullOrEmpty(value))
					{
						separator = value;
					}
				}
		}

Reply:
{userfiles:$}file1$file2
{userfiles:@@@}file1@@@file2

Reply:
I personally think it is overly complicated, at least until we establish it is something truly necessary.
A `;` as a separator is a good choice IMHO.

Why does it have a space?

Reply:
> I personally think it is overly complicated, at least until we establish it is something truly necessary.
A ; as a separator is a good choice IMHO.

I agree. 

> Why does it have a space?

It's more readable in the textbox, than just ";".
"C:/SomeDir/SomeFile;C:/SomeDir.OtherFile;C:/SomeDir/SomeOtherFile" vs. "C:/SomeDir/SomeFile; C:/SomeDir.OtherFile; C:/SomeDir/SomeOtherFile"